restful-tango/tangoREST.py

```diff
@@ -137,9 +137,11 @@ def convertJobObj(self, dirName, jobObj):
 		# List of input files
 		input = []
 		for file in jobObj['files']:
+            inFile = file['localFile']
+            vmFile = file['destFile']
```

Comment:
[PEP 8][1] Says that spaces are preferred.

[1]: https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces